RFUSB_to_DB15/drivers.h

```diff
@@ -44,6 +46,8 @@
 #define PID_HORI_LITE           0x00EE // HORI ワイヤードコントローラライト for PS4-102
 #define PID_HORI_RAP_PREMIUM    0x0026 // HORI Real Arcade リアルアーケードPro.3 Premium VLX
 #define PID_HORI_RAP_PS3        0x0011 // HORI Real Arcade Pro.3 SA PS3コントローラ
+#define PID_DAEMON_SNES         0x8036 // DaemonBite Retro Controllers SNES
```

Comment:
This is also the default PID for an Ardiuno. Maybe we should include that in the comment?